File: despatch.WEB/app/src/swagger/models/contact-data-phone-numbers-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { ContactDataRequest } from './contact-data-request';
import { PhoneNumberRequest } from './phone-number-request';
export interface ContactDataPhoneNumbersRequest extends BaseEntityRequest{
  contactData?: ContactDataRequest;
  contactDataId?: string;
  phoneNumber?: PhoneNumberRequest;
  phoneNumberId?: string;
}

File: despatch.Domain/Models/Default/Request/ContactDataPhoneNumbersRequest.cs
namespace despatch.Domain.Models.Default.Request
{
    public class ContactDataPhoneNumbersRequest : BaseEntityRequest
    {
        public string ContactDataId { get; set; }
        public string PhoneNumberId { get; set; }

        public virtual ContactDataRequest ContactData { get; set; }
        public virtual PhoneNumberRequest PhoneNumber { get; set; }
    }
}
